ASR/recognition/README.md

```diff
@@ -0,0 +1,809 @@
+
+
+
+
+
+
+<!DOCTYPE html>
```

Comment:
Despite the filename `README.md`, this is an HTML, not markdown, file. It looks like you saved the web page of GitHub rendering your markdown file, which artificially inflates the size of this file. Try viewing the markdown source with the link to the "raw" file.

Reply:
Thanks Jonathan, sorry I'm new to github. I'll do that.